pipeline/docs/ess-dive-submissions.md

```diff
@@ -0,0 +1,38 @@
+
+# Steps to update ESS-DIVE submissions
+
+0.  Run processing pipeline to produce L1 data for synoptic and TEMPEST sites
+
+1.  Move L1 data to folders labeled `vX-X` \> `vX-X_SYN` and `vX-X` \> `vX-X_TMP` . Each folder gets a copy of the `READMD.txt` and `README_vX-X.txt` files.
+
+2.  Prep package metadata files
+
+-   Produce flmd using `flmd-generator.R`
```

Comment:
Is it obvious how to do this? Do you need to point the script to where the L1 data are?

Reply:
Good question, I think currently flmd-generator requires the files to be in the L1 folder (at least i havent tried otherwise) which would impact the steps here